sdk/examples/wallet/output_consolidation.rs

```diff
@@ -12,6 +12,7 @@ use iota_sdk::{
         constants::SHIMMER_COIN_TYPE,
         secret::{mnemonic::MnemonicSecretManager, SecretManager},
     },
+    types::block::address::ToBech32Ext,
```

Comment:
Just asking again here as the discussion on Slack didn't continue, but can you elaborate on why this is better? Sounds a tiny bit more annoying UX wise

Reply:
Covers all the address variants rather than just `Address`.